src/main/java/com/epam/ta/reportportal/entity/oauth/OAuthRegistrationRestriction.java

```diff
@@ -0,0 +1,78 @@
+package com.epam.ta.reportportal.entity.oauth;
+
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Objects;
+
+@Entity
+@EntityListeners(AuditingEntityListener.class)
+@Table(name = "oauth_registration_restriction", schema = "public")
+public class OAuthRegistrationRestriction implements Serializable {
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Column(name = "id", unique = true, nullable = false, precision = 64)
+	private Integer id;
```

Comment:
Replace Integer id type with Long

Reply:
Remove precision

Reply:
you could remove all these restrictions as unique and nullable cause we decided leave all that on db side